ws-service/server.ts

```diff
@@ -47,22 +47,35 @@ const addressRouteConnection = (socket: Socket): void => {
 const broadcastTxs = async (broadcastTxData: BroadcastTxData): Promise<void> => {
```

Comment:
This is changing the whole endpoint to take out information from the args is receiving, and you're not chaning how this information is created when this is broadcasted (in services/chronikService.ts, for example). So we're creating data there that just right way gets thrown out. Why not make this expected the simplified type already?

Reply:
because this is used in some other parts of the logic so I decided to focus on changing the data that goes out